README.md

```diff
@@ -56,11 +56,18 @@
 2. `tar -zxvf YOUR_FILE.tar.gz`
 3. Add the now-extracted `./3mux` to your `$PATH`
 
+###### Using Nix flakes
```

Comment:
It might be useful to say that this is for Nix 2.4 and up since on Nix 2.3, it's experimental